vendors/renesas/boards/rx65n-rsk/aws_demos/config_files/aws_demo_config.h

```diff
@@ -46,7 +46,7 @@
  *
  *  These defines are used in iot_demo_runner.h for demo selection */
 
-#define CONFIG_CORE_MQTT_MUTUAL_AUTH_DEMO_ENABLED
+#define CONFIG_OTA_MQTT_UPDATE_DEMO_ENABLED
```

Comment:
The default demo should be core mqtt mutual auth demo , is this intentional?

Reply:
Changed it to core mqtt mutual auth.